### PR TITLE
Inspection error handling

### PR DESCRIPTION
**Pull request by aomerk**

### CODE REVIEW

File: inspect/trace_api.go [hunk 1]
```diff
@@ -61,6 +68,7 @@ func (tai *TraceAPIInspector) Inspect(ctx context.Context, inspectionCfg Inspect
 	if err != nil {
 		resultErr = multierror.Append(resultErr, fmt.Errorf("failed to get chain id: %w", err))
 		results.Indicators[IndicatorTraceAccessible] = ResultFailure
+		results.Indicators[IndicatorTraceSupported] = ResultFailure
 	} else {
 		results.Indicators[IndicatorTraceAccessible] = ResultSuccess
```
Comment (canercidam) [inline inspect/trace_api.go:73]: Do we need to set "trace supported" as successful here?

**Review by canercidam — COMMENTED** [review]
LGTM! Just one thing caught my eye.

File: inspect/trace_api.go (hunk 1 above)
Comment (aomerk) [reply inspect/trace_api.go:73]: Did you mean "as failure"?

Comment (canercidam) [reply inspect/trace_api.go:73]: Never mind - it makes sense

**Pull request merged**